canonical-web-and-design/vanilla-framework
# Add the chip component

## Done
Add the chip component

Fixes https://github.com/canonical-web-and-design/vanilla-framework/issues/3143

## QA
- Pull code
- Run `./run`
- Open http://0.0.0.0:8101/docs/patterns/chip
- Check the content is ok
- Open http://0.0.0.0:8101/docs/examples/patterns/chip
- Check the component renders as expected
